lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -184,7 +184,15 @@ export const BrowserAuthErrorMessage = {
     nativePromptNotSupported: {
         code: "native_prompt_not_supported",
         desc: "The provided prompt is not supported by the native platform. This request should be routed to the web based flow."
-    }
+    },
+    accessCodeNotInCache: {
```

Comment:
```suggestion
    accessTokenNotInCache: {
```

Nit: For consistency, I suggest "token" instead of "code"